litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml

```diff
@@ -249,47 +249,53 @@
-            status_code: 200
-          register: node3
-          failed_when: "node3.json.nodeType['mayadata.io/data-plane'] != 'true'"
+        ## Checking the control plane label on the first node                                                                                                                                                          
+        - name: Checking the label on the first node of the cluster                                                                                                                                                    
+          uri:                                                                                                                                                                                                          
+            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[0] }}"                                                                                                                                   
+            method: GET                                                                                                                                                                                                 
+            url_username: "{{ username.stdout}}"                                                                                                                                                                        
+            url_password: "{{ password.stdout}}"                                                                                                                                                                        
+            force_basic_auth: yes                                                                                                                                                                                       
+            return_content: yes                                                                                                                                                                                         
+            body_format: json                                                                                                                                                                                           
+            status_code: 200                                                                                                                                                                                            
+          register: node1                                                                                                                                                                                               
+          until: "'mayadata.io/control-plane' in {{node1.json.labels | to_json | string }}"                                                                                                                                
```

Comment:
@amitbhatt818 I guess this condition will now work fine without converting the node1 json. Can you please verify this once?

Reply:
Yes @harshshekhar15 It is working fine without json also.